### DISCUSSION TITLE
Is M110.0 expected to be launched next week? and HTML5 Video?

### DISCUSSION THREAD

**Discussion by whxuezhe** [Alex313031/Thorium-Win-AVX2#53] [General]
Chrome Canary 110.0 has introduced the Html5 Video experimental feature, based on the WebVTT audio and video parsing code framework. Will our thorium build browser follow up and introduce it in time? , and the approximate push time of our core M110.0 will not be later than the end of next week, right? .

**Comment by gz83** [comment]
> Chrome Canary 110.0 has introduced the Html5 Video experimental feature, based on the WebVTT audio and video parsing code framework. Will our thorium build browser follow up and introduce it in time? , and the approximate push time of our core M110.0 will not be later than the end of next week, right? .

@whxuezhe Hello, thank you for your support of Thorium browser, Thorium will be updated in the near future, please be patient and pay attention to Thorium repo
